frontend/src/v5/store/tickets/tickets.redux.ts

```diff
@@ -71,7 +71,11 @@ export const replaceTemplateSuccess = (state: ITicketsState, { modelId, template
 	const modelTemplate = state.templatesByModelId[modelId]
 		.find((loadedTemplate) => loadedTemplate._id === template._id);
 
-	mergeWithArray(modelTemplate, template);
+	if (modelTemplate) {
+		mergeWithArray(modelTemplate, template);
+	} else {
+		state.templatesByModelId[modelId].push(template);
```

Comment:
I'm not sure I fully understand. Do you mean if a template is made that doesn't have properties, modules, and config? If so:

1. I'm not sure how common it would be (it's not a very useful ticket)
2. I'm not sure how you'd determine that there is nothing more to fetch without first doing a fetch
3. Since this is only one call when opening up ticket details I don't think it's that demanding

Oh, unless you mean if the state already includes B properties then there is no need to make the call again when opening/creating a ticket with that template. That's actually a good point

Reply:
yep, the last bit is exactly what I am referring to